packages/neo-one-smart-contract-lib/src/ownership/Secondary.ts

```diff
@@ -0,0 +1,33 @@
+import { Address, constant, createEventNotifier, SmartContract } from '@neo-one/smart-contract';
+
+/**
+ * @title Designated Caller
+ * @dev This contract can only be called by 1 other address.
+ */
+const primaryTransferred = createEventNotifier<Address, Address>('primary_transferred', 'from', 'recipient');
+
+export class Secondary extends SmartContract {
+  protected mutablePrimary: Address;
```

Comment:
This should probably use the same scheme as `Ownable` in terms of setting the initial primary. Also, any reason why this is called `mutablePrimary` (and `primary` used throughout) rather than `secondary`?

Reply:
@dicarlo2 

RE: Secondary vs Primary: This contract *is* secondary to the Primary contract.  This follows the model here:  https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/ownership/Secondary.sol

Reply:
Updated initialPrimary scheme to be like Ownable.